entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/response.proto

```diff
@@ -41,4 +41,13 @@ message DeleteEntitiesResponse {
 }
 
 message BulkUpdateAllMatchingFilterResponse {
+  repeated UpdateSummary summaries = 1;
+}
+
+message UpdateSummary {
+  repeated UpdatedEntity responses = 1;
+}
+
+message UpdatedEntity {
+  string id = 1;
```

Comment:
sorry poorly worded my other comment, I meant entity types on the response (since they can be mixed, right?). I don't think we need the double-repeated:
```suggestion
  repeated UpdatedEntity updated_entities = 1;
}

message UpdatedEntity {
  string type = 1;
  string id = 2;
```

Reply:
> since they can be mixed, right?

Nope. All the updates should be of the same entity type. Whereas different update operations are supported based on different filters.

We need 2 repeated because the top level is for the number of update operations and the second level is for the actual entity ids that got updated. Since each update operation could update multiple entities, we need a repeated there also.
```
message BulkUpdateAllMatchingFilterRequest {
  string entity_type = 1;
  repeated Update updates = 2;
}

message Update {
  Filter filter = 1;
  repeated AttributeUpdateOperation operations = 2;
}
```

Reply:
got it.

Reply:
got it. seems fine to me then